src/lxc/seccomp.c

```diff
@@ -550,6 +547,25 @@ bool do_resolve_add_rule(uint32_t arch, char *line, scmp_filter_ctx ctx,
 	return true;
 }
 
+struct scmp_ctx_info {
+	uint32_t architectures[3];
+	enum lxc_arch_t lxc_arch[3];
+	scmp_filter_ctx contexts[3];
+	bool needs_merge[3];
+};
+
+static int get_arch_index(enum lxc_arch_t arch, struct scmp_ctx_info *ctx)
+{
+	int i;
+
+	for (i = 0; i < 3; i++) {
+		if (ctx->lxc_arch[i] == arch)
+			return i;
+	}
```

Comment:
Hm, this could probably be a define:
```c
#define SCMP_ARCH_INDEX_MAX 3
```
and then
```c
for (i = 0; i < SCMP_ARCH_INDEX_MAX; i++)
        if (ctx->lxc_arch[i] == arch)
                return i;
```

Reply:
Fixed